### ISSUE TITLE
"Applied Accessibility - Make Screen Reader Navigation Easier with the footer Landmark" cannot be completed because it always thinks there are <div>s

### ISSUE THREAD

**Issue by SuperGeniusZeb** [freeCodeCamp/learn#130]
https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark

This lesson cannot be completed because one of the tests ("Your code should not have any `div` tags.") always fails, no matter what the content of the editor is.

Also, there are 2 duplicate `div` tests, unlike the previous 3 lessons.

Related:
- #127
- #128
- #129
- #134
- #136

**Comment by Bouncey** [comment]
I cannot reproduce, can you share your code please?

**Comment by SuperGeniusZeb** [comment]
```
<body>
  <header>
    <h1>Training</h1>
    <nav>
      <ul>
        <li><a href="#stealth">Stealth &amp; Agility</a></li>
        <li><a href="#combat">Combat</a></li>
        <li><a href="#weapons">Weapons</a></li>
      </ul>
    </nav>
  </header>
  <main>
    <section id="stealth">
      <h2>Stealth &amp; Agility Training</h2>
      <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
      <article><h3>No training is NP-complete without parkour</h3></article>
    </section>
    <section id="combat">
      <h2>Combat Training</h2>
      <article><h3>Dispatch multiple enemies with multithreaded tactics</h3></article>
      <article><h3>Goodbye world: 5 proven ways to knock out an opponent</h3></article>
    </section>
    <section id="weapons">
      <h2>Weapons Training</h2>
      <article><h3>Swords: the best tool to literally divide and conquer</h3></article>
      <article><h3>Breadth-first or depth-first in multi-weapon training?</h3></article>
    </section>
  </main>


  <footer>&copy; 2016 Camper Cat</footer>


</body>
```
The `div` test also fails if there is nothing in the editor.

I just tested in Chromium, and the tests pass when using that browser. (However, there are still duplicate `div` tests.) Can you reproduce the issue in Firefox?

Also, I have noticed that the website switched back and forth between 2 designs yesterday. Does the website look like this to you right now?
![image](https://user-images.githubusercontent.com/19592990/40842621-153344d2-6574-11e8-95a4-9871f4040928.png)

(The other design has a second row under the header with the <kbd><samp>Run the Tests</samp></kbd> button in it, and the tests show up under the preview of the project instead of the description.)

**Comment by SuperGeniusZeb** [comment]
I should have clarified, but I was using Firefox Nightly. I recently switched to Firefox Beta, and the issue is still occurring there. I am using Antergos Linux.

I tried disabling various Firefox add-ons, and it turns out that Plasma Integration is the add-on causing the issue:

https://addons.mozilla.org/en-US/firefox/addon/plasma-integration/

Disabling the add-on makes the `<div>` tests immediately start working without even refreshing the lesson page. Likewise, enabling the add-on immediately causes the `<div>` tests to stop working.

I have no idea why this add-on causes an issue. This issue only seems to happen with tests for `<div>`s, not any other HTML element.

The corresponding Chromium extension does not cause any issues with the freeCodeCamp tests. I have no idea if it is because of a difference between Firefox and Chromium, a difference between the Firefox add-on and its matching Chromium extension, both, or neither.

**Comment by raisedadead** [comment]
https://github.com/freeCodeCamp/learn/issues/136#issuecomment-398936869
